language/tools/move-cli/src/main.rs

```diff
@@ -220,6 +221,7 @@ fn run(
         })
         .collect();
 
+    let logger = StdErrLogger;
```

Comment:
I suppose this is a question for @sblackshear: is a std error logger a good solution for the cli?

Reply:
StdErrLogger sounds great. I am planning to add a `quiet` mode in the future, but I can choose `NoLogLogger` instead in that mode.